app/models/resources.js

```diff
@@ -0,0 +1,68 @@
+/* eslint-disable max-len */
```

Comment:
I would like to have resource model and resource data separation, i.e.

* resourceData.js => data that will be updated by copy-grand-masters
* resourceModel.js => methods for operating on that data, `findByKeyword, getById`